io/net/bonding.py

```diff
@@ -521,6 +528,7 @@ def test_cleanup(self):
         clean up the interface config
         '''
         self.bond_remove("local")
+        detected_distro = distro.detect()
```

Comment:
@FarooqAbdulla02 can you make this as a global variable and use where ever it is needed.

Reply:
@Naresh-ibm updated the code by fixing as global variable.